git_machete/client.py

```diff
@@ -1690,7 +1691,16 @@ def checkout_github_prs(self, pr_no: int) -> None:
         if not pr:
             raise MacheteException(f"PR #{pr_no} is not found in repository `{org}/{repo}`")
         if '/'.join([remote, pr.head]) not in self.__git.get_remote_branches():
-            raise MacheteException(f"Could not check out PR #{pr_no} because its head branch `{pr.head}` is already deleted from `{remote}`.")
+            remote_from_pr: str = pr.repo['full_name'].split('/')[0]
```

Comment:
Is it possible that repo would have `/` in the name? Like `first_part_of_remote_name/second_part_of_remote_name` and than branch from remote would look like that `first_part_of_remote_name/second_part_of_remote_name/my_branch`

Reply:
This is how Github API outputs, under name `full_name` its rather organisation/repo_name. Since i want to add new remote here, `organisation` fits more, because it might happen that there would be 2 PRs from forks at the same time, so adding remote name same to repository name will cause unexpected behaviour, cause one of the remotes will not be added then.